.rosinstall

```diff
@@ -0,0 +1,4 @@
+- git: { local-name: tork-a/spur, uri: 'http://github.com/tork-a/spur.git', version: master }
+- git: { local-name: ccny-ros-pkg/scan_tools, uri: 'https://github.com/ccny-ros-pkg/scan_tools.git', version: master }
+- git: { local-name: ros-teleop/teleop_twist_joy, uri: 'https://github.com/130s/teleop_twist_joy.git', version: add/omnidir }
```

Comment:
> please check if deb version works, latest version includs ros-teleop/teleop_twist_joy#6

I'll leave this for future pull request for now; I won't have an access to joystick device (in particular the one from Elecom [as instructed in readme](https://github.com/tork-a/spur/blob/d412f99ceef0c4cf49bb3a18315e5e9743f3bb3b/README.rst#teleop-with-joystick-with-elecom-ps3)) so that I can't test the deb.